velox/docs/functions/presto/conversion.rst

```diff
@@ -43,6 +43,7 @@ supported conversions to/from JSON are listed in :doc:`json`.
      - double
      - varchar
      - timestamp
+     - timestamp with time zone
```

Comment:
I believe the cast logic is not very straightforward and therefore is worth documenting. Specifically, I believe the results of the case depend on configuration properties. Please, clarify.

Reply:
Thanks for the prompt review. I've updated the doc with details. Let me know if you'd like to adjust anything.